.github/workflows/ci.yml

```
@@ -194,18 +195,22 @@ jobs:
     if: startsWith(github.head_ref, 'master') || github.event.action == 'ready_for_review' || !github.event.pull_request.draft
     uses: ./.github/workflows/examples.yml
     with:
-      ANSYS_VERSION: '241'
+      ANSYS_VERSION: "241"
```

Can you use the environment variable?
```suggestion
      ANSYS_VERSION: ${{env.ANSYS_VERSION}}
```